src/ColumnsTraits/ManageColumns.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+
+namespace Backpack\CRUD\ColumnsTraits;
+
+use Backpack\CRUD\Columns\Column;
+use Backpack\CRUD\Facades\CRUDPanel;
+
+trait ManageColumns
+{
+
+    /**
+     * Remove all previously set columns and add the provided ones
+     * @param $columns array
+     */
+    public function set($columns)
+    {
+        CRUDPanel::setColumns($columns);
+    }
+
+    /**
+     * @param $columns [string, array or multidimensional array]
+     * @return \Backpack\CRUD\Facades\CRUDPanel
+     */
+    public function add($columns)
+    {
+        // if a string was passed
+        if (! is_array($columns)) {
```

Comment:
You want `is_string` - that could be a `OliverZieglerRocks` object for all we know!